test/integration_test.py

```diff
@@ -904,12 +914,14 @@ def update(source: SourceRef,
                                                                   prefix))
 
         list(starmap(update, self._list_managed_access_bundles(catalog)))
-        num_bundles = self.min_bundles - len(bundle_fqids)
+        # Ensure that at least one public source will be indexed
```

Comment:
This comment seems a little far away from the actual measure that's being taken, which is passing `public_1st=True`. It also should explain why the measure is taken. I usually explain what would happen if I didn't take the measure.